scripts/merge_models

```diff
@@ -0,0 +1,41 @@
+#%%
+from ribasim_nl import CloudStorage
+import geopandas as gpd
+import pandas as pd
+from pathlib import Path
+
+storage = CloudStorage()
+out_gpkg = Path(r"D:\repositories\hydrobase\data\rijkswateren\database.gpkg")
```

Comment:
This probably shouldn't be an absolute path.

Reply:
File removed